lib/wallet/test/unit/Cardano/Wallet/Shelley/Compatibility/LedgerSpec.hs

```diff
@@ -56,6 +66,17 @@ spec = describe "Cardano.Wallet.Shelley.Compatibility.LedgerSpec" $
         ledgerRoundtrip $ Proxy @TokenQuantity
         ledgerRoundtrip $ Proxy @TxIn
 
+    describe "Timelock roundtrips" $ do
+        let ledger = toLedgerTimelockScript @StandardBabbage
+        let wallet = toWalletScript (const Unknown)
+
+        it "toWalletScript . toLedgerTimelockScript == id" $ property $ \s -> do
+            -- Ignore key role by doing one extra conversion
+            ledger (wallet $ ledger s) === ledger s
```

Comment:
Hi @Anviking.

The name of the property is inconsistent with the actual property being tested. I don't think we should do that!

I'm not sure this is a good method of naming properties, since:
- If we make the description string match the property, it will easily overflow the line limit.
- If we add a line break, it then becomes hard to search the code for the property name.
- It creates an artificial limit on the length of a property.
- It creates an opportunity for bit rot -- someone can update the property, but forget to update the description.

Instead of writing these properties in line, perhaps we can extract them out to separate functions instead?

Alternatively:
```suggestion
        it "ledger . wallet . ledger == ledger"
            $ property $ \s ->
                -- Ignore key role by doing one extra conversion
                (ledger . wallet . ledger) s === ledger s
```